server/conf/evolutions/default/83.sql

```diff
@@ -0,0 +1,31 @@
+# --- !Ups
+
+ALTER TABLE applications ADD COLUMN IF NOT EXISTS status_create_time TIMESTAMP;
```

Comment:
(blocking) Same thing here. `last_status_changed_time` better conveys the purpose of the column.

Reply:
hmm.. last_status_changed_time feels a little confusing. This column is populated even when a default status is applied to the application, that why I picked status_create_time to denote- when the current status was created on the application. How does "status_applied_time" sound to you?